mlptrain/potentials/mace/mace.py

```diff
@@ -0,0 +1,556 @@
+import numpy as np
+from time import time
+from copy import deepcopy
+from mlptrain.box import Box
+from mlptrain.log import logger
+from mlptrain.config import Config
+from mlptrain.potentials._base import MLPotential
+import ast
+import logging
+import os
+from typing import Optional
+import torch.nn.functional
+from e3nn import o3
+from torch.optim.swa_utils import SWALR, AveragedModel
+from torch_ema import ExponentialMovingAverage
+import mace
+from mace import data, modules, tools
+from mace.tools import torch_geometric,  evaluate
+from mlptrain.config import Config
+from ase.data import chemical_symbols
+import dataclasses
+from prettytable import PrettyTable
+import torch
+
+
+@dataclasses.dataclass
+class SubsetCollection:
+    train: data.Configurations
+    valid: data.Configurations
+
+class MACE(MLPotential):
+
+    def _train_file(self,
+                    valid_fraction = 0.1,
+                    config_type_weights = '{"Default":1.0}' ,
+                    model = 'MACE',
+                    loss = 'weighted',
+                    hidden_irreps = '128x0e + 128x1o',
+                    batch_size = 10,
+                    r_max=5 ,
+                    correlation = 3,
+                    device = 'cuda',
+                    error_table = 'TotalMAE',
+                    swa = True,
+                    #start_swa= 1200,
+                    ema = False,
+                    ema_decay=0.99,
+                    amsgrad = True,
+                    #max_num_epochs=800,
+                    restart_latest = True,
+                    save_cpu = True
+                    ) -> None:
+        """
+        Train this potential on the current training data is adjusted from run_train.py in mace package
```

Comment:
"Train this potential on the current training data is adjusted" - doesn't make sense to me